projects/Rockchip/packages/retrorun/package.mk

```diff
@@ -2,32 +2,44 @@
 # Copyright (C) 2021-present Shanti Gilbert (https://github.com/shantigilbert)
```

Comment:
Also please add your copyright information to the header (makes it easier to know who worked on which file)